src/test/scala/net/ruippeixotog/scalascraper/Examples.scala

```diff
@@ -17,6 +18,14 @@ object ExampleMatchers {
   val errs = validatorsAt[Int]("error-matchers")
 }
 
+object ProxyApp extends App {
+  ProxyUtils.setProxy("localhost", 3128)
+  val browser = Browser()
+  val doc = browser.get("http://observador.pt")
+
+  println("=== OBSERVADOR HTTP & HTTPS PROXY ===")
```

Comment:
Ok, looks good to me then! As I do not have a proxy readily available to test, I wasn't sure that doing a GET request would be enough.